### PR TITLE
Update list of developers

### PR DESCRIPTION
**Pull request by kspieks**
This PR updates the list of current developers on RMG website: http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/credits.html#how-to-cite

I added all current members that do work related to RMG. I apologize in advance if I missed anyone. 

@sevyharris could you please add people from Professor West's group to help us update the list?

### CODE REVIEW

**Review by rwest — DISMISSED** [review]
I added our group's names (and some of Franklin's). Just needs rebasing and merging, I think.

**Comment by kspieks** [comment]
Thanks! I just rebased

**Review by JacksonBurns — APPROVED** [review]
This does not change any source code, can merge w/o waiting for tests.

File: documentation/source/users/rmg/credits.rst [hunk 1]
```diff
@@ -4,7 +4,9 @@
 Credits
 *******
  
-RMG is based upon work supported by the Department of Energy, Office of Basic Energy Sciences through grant DE-FG02-98ER14914 and by the National Science Foundation under Grants No. 0312359 and 0535604.
+RMG is based upon work supported by the Department of Energy, Office of Basic Energy Sciences through grant DE-FG02-98ER14914 and  as part of the Computational Chemical Sciences Program. 
+and by the National Science Foundation under Grants No. 0312359, 0535604, and 1751720.
```
Comment (hwpang) [inline documentation/source/users/rmg/credits.rst:8]: Weird line change? Should these two be in the same sentence?

**Review by hwpang — CHANGES_REQUESTED** [review]
Just one comment!

File: documentation/source/users/rmg/credits.rst (hunk 1 above)
Comment (hwpang) [reply documentation/source/users/rmg/credits.rst:8]: @rwest Do you know if the second line should be removed, or should they be combined with the first line? I'm not familiar with the RMG funding source situation.

**Pull request merged**